content/docs/command-reference/remote/modify.md

```diff
@@ -154,6 +154,14 @@ these settings, you could use the following options.
   $ dvc remote modify myremote secret_access_key my-secret_access_key
   ```
 
+- `session_token` - AWS MFA Session Token. May be used (along with
+  `access_key_id` and `secret_access_key`) instead of `credentialpath` when MFA
+  is required:
+
+  ```dvc
+  $ dvc remote modify myremote session_token my-session-token
```

Comment:
let's make it `--local` I think. Those tokens are not permanent, they should not be going into the regular Git-controlled config

Reply:
Good point although we don't use `--local` much in this doc, in particular we don't use it at all in the S3 sections... I guess I should update that in some misc. update later.

Reply:
@jorgeorpinel please, let's create a ticket, let's not focus on fixing it now

Reply:
Sure, done: #2089